### ISSUE TITLE
Marker testID prop not visible as accessibility identifier

### ISSUE THREAD

**Issue by ja-ka** [react-native-maps/react-native-maps#3622]
# Bug report

## Summary

The defined testID property on a Marker gets lost. It is not visible as the accessibility identifier when running the app on the iOS simulator and inspecting the view hierarchy. Therefore i.e. e2e testing using detox is not possible.

## Environment info

react: 16.13.1
react-native: 0.63.3
react-native-maps: 0.27.1

I'm using AppleMaps (DEFAULT_PROVIDER)

`react-native info` output:

```bash
System:
    OS: macOS 11.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 171.25 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.12.0 - /usr/local/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.9.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.0, DriverKit 20.0, macOS 11.0, tvOS 14.0, watchOS 7.0
    Android SDK:
      API Levels: 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.0
      System Images: android-28 | Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom_64, android-30 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.0 AI-193.6911.18.40.6514223
    Xcode: 12.0/12A8169g - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_261 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.13.1 => 16.13.1 
    react-native: 0.63.3 => 0.63.3 
  npmGlobalPackages:
    *react-native*: Not Found
```

## Steps to reproduce

```bash
$ git clone https://github.com/ja-ka/react-native-maps-marker-test-id.git
$ cd react-native-maps-marker-test-id
$ yarn install
$ npx pod-install
$ open ios/MarkerTestIDExample.xcworkspace/
```

Build the project and check the hierarchy debug view.

Describe what you expected to happen:

When selecting one of the two markers in the hierarchy debug view, the accessibility identifier should contain the value `marker`.

Describe what actually happens:

Defined testID property is not visible in the view hierarchy in Xcode, the accessibility identifier is `null`.

## Reproducible sample code

https://github.com/ja-ka/react-native-maps-marker-test-id

**Comment by vitalyiegorov** [comment]
Still reproducible.

**Comment by anija** [comment]
Any news on this?

**Comment by angelica-snowit** [comment]
Why was this bug closed?
